CMakeLists.txt

```diff
@@ -200,16 +200,25 @@ else()
 endif()
 
 message(STATUS "Uncompressing muparserx static library...")
-if(MSVC)
-	set(PLATFORM "win64")
-elseif(APPLE)
-	set(PLATFORM "macos")
-elseif(UNIX)
-	set(PLATFORM "linux")
-endif()
-execute_process(COMMAND ${CMAKE_COMMAND} -E tar "xvfj" "${AER_SIMULATOR_CPP_SRC_DIR}/third-party/${PLATFORM}/lib/muparserx.7z"
-	WORKING_DIRECTORY  "${AER_SIMULATOR_CPP_SRC_DIR}/third-party/${PLATFORM}/lib/")
-set(MUPARSERX_LIB_PATH "${AER_SIMULATOR_CPP_SRC_DIR}/third-party/${PLATFORM}/lib")
+#if(MSVC)
```

Comment:
Ok, just remove all these commented lines. We can always go back in git history to see this code.

Reply:
Yeah! I forgot!